.github/workflows/README.md

```diff
@@ -31,11 +44,59 @@ To run a full build and download all datasets, you would then combine both tags:
 $ git commit -m "[full doc][force download] request full build"
 ```
 
-## Skip CI
+#### Skip CI
 
 You can decide to skip documentation building and tests execution at any time by including the tag "[skip ci]" in your commit message.
 For more information, see: https://docs.github.com/en/actions/managing-workflow-runs/skipping-workflow-runs
 
 ```bash
 $ git commit -m "[skip ci] commit message"
 ```
+
+### trigger-hosting.yml
+
+Runs only if the workflow in `build-docs.yml` completes successfully. Triggers the CircleCI job described below.
+
+## Hosting and deploying development documentation
+
+### [.circleci/config.yml](/.circleci/config.yml)
+
+Artifacts hosting and deployment of development docs use CircleCI. See [.circleci/README.md](/.circleci/README.md) for details.
+On a pull request, only the "host" job is run. Then the artifacts can be accessed from the `host_and_deploy_doc` workflow seen under the checks list. Click on "Details" and then on the "host_docs" link on the page that opens. From there you can click on the artifacts tab to see all the html files. If you click on any of them you can then normally navigate the pages from there.
+With a merge on main, both "host" and "deploy" jobs are run.
+
+## Quick link to CircleCI artifacts (BROKEN)
```

Comment:
Unclear to me what this BROKEN is?

Reply:
I put that there because it doesn't work. But I agree it's a bit weird. I'll instead leave it out until we get it fixed.